packages/babel-types/src/modifications/flow/removeTypeDuplicates.ts

```diff
@@ -48,7 +49,8 @@ export default function removeTypeDuplicates(
 
     if (isUnionTypeAnnotation(node)) {
       if (typeGroups.indexOf(node.types) < 0) {
-        nodes.push(...node.types);
+        // todo(babel-8): remove type casting
+        (nodes as any).push(...node.types);
```

Comment:
This is still a breaking change: it's mutating a public parameter which is not supposed to be mutated.

Reply:
Wasn't it the same before this PR?

Reply:
There was concat before, which doesn't modify the original array.

Reply:
Hello is this configuration gonna work?